server/src/main/java/org/elasticsearch/common/logging/Loggers.java

```diff
@@ -49,16 +49,17 @@
         Setting.prefixKeySetting("logger.", (key) -> new Setting<>(key, Level.INFO.name(), Level::valueOf, Setting.Property.Dynamic,
             Setting.Property.NodeScope));
 
-    public static Logger getLogger(Class<?> clazz, Settings settings, ShardId shardId, String... prefixes) {
-        return getLogger(clazz, settings, shardId.getIndex(), asArrayList(Integer.toString(shardId.id()), prefixes).toArray(new String[0]));
+    public static Logger getLogger(Class<?> clazz, ShardId shardId, String... prefixes) {
+        return getLogger(clazz, Settings.EMPTY,
```

Comment:
I'll make follow up changes that remove the need for `Settings.EMPTY` eventually.

Reply:
They are just needed to make it compile and aren't used.